apps/web/src/components/Header.tsx

```diff
@@ -1,13 +1,16 @@
 import Link from "next/link";
 import Image from "next/image";
 import logoDarklake from "../../public/images/logo-h-darklake.png";
+import ConnectedWalletDropdown from "./ConnectedWalletDropdown";
 
 const Header = () => {
+  const currentWalletAddress = null;
```

Comment:
make sure to wrap these with note/todo comment

```suggestion
  // TODO: wire with added/connected wallet
  const currentWalletAddress = null;
```